### PR TITLE
feature(angular-v17): Query the blog 'favicon' or 'logo' or 'author image' in header in v17 native

### LINKED ISSUE #280: Query the blog 'favicon' or 'logo' or 'author image' in header in v17 native
# Angular App version

- [x] `angular-v17` - has no UI libraries
- [ ] `angular-v17-modMAT` - modules and Angular Material
- [ ] `angular-v17-AnguMAT` - has Angular Material
- [ ] `angular-v17-PrimeNG` - has PrimeNG

**Is your feature request related to a problem? Please describe.**
In the header in the logo - right now the image of the project is loaded and as we discussed 
the intension is to use the image for the actual blog of whoever is using the app. 

**Describe the solution you'd like**
So there are few possible ways to do this:
- first perform check if there is a `favicon` is yes - use it
- if no favicon, then check for publication logo
- if no logo then query author image

**Additional context**
Requires a slight addition to the already existing blog info query 

edit by @cleancodecraft 
# adding screenshots to every issue is not a common practice in open source, 
- reading the description, 
- looking at the lables 
- asking question BEFORE submitting a PR is the responsibility of the contributor 

This is what this written above

# `Requires a slight addition to the already existing blog info query ` - if you do not understand what it means then maybe don't say you have closed the issue ?

image

The image to be replaced 

image

### LINKED ISSUE #338: Add a blog `favicon` DOM element in in the `<head>` section of the angular-v17 app version
# Angular App version

- [x] `angular-v17` - has no UI libraries
- [ ] `angular-v17-modMAT` - modules and Angular Material
- [ ] `angular-v17-AnguMAT` - has Angular Material
- [ ] `angular-v17-PrimeNG` - has PrimeNG

**Additional context**
Query made and blog image or author profile image is set as site favicon in the `<head>` section 
- the `href` of the link is set with conditional checks

### PR DESCRIPTION
**Pull request by jhonacs2**
# Angular App version

- [x] `angular-v17` - has no UI libraries
- [ ] `angular-v17-modMAT` - modules and Angular Material
- [ ] `angular-v17-AnguMAT` - has Angular Material
- [ ] `angular-v17-PrimeNG` - has PrimeNG

## This PR Closes Issue 
closes #338 

## Description
Query the blog 'favicon' or 'logo' or 'author image' in header in v17 native

## What type of PR is this? (check all applicable)
- [x] 🅰️ Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added to documentation?
- [ ] 📜 README.md
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by cleancodecraft** [comment]
Details are adjusted to reflect the actual activity done in this PR

**Review by cleancodecraft — APPROVED** [review]

**Review by monacodelisa — APPROVED** [review]

File: angular-v17/src/app/models/blog-info.ts [hunk 1]
```diff
@@ -19,3 +21,6 @@ export type SocialLinks = {
   mastodon: string;
 };
 
+export interface Author {
```
Comment (cleancodecraft) [inline angular-v17/src/app/models/blog-info.ts:24]: @jhonacs2 did you not see that the Author interface is already listed in post.ts ?

**Comment by cleancodecraft** [comment]
@monacodelisa this doesn't actually even solve the favicon in the `<head>` section as a DOM element, please look into it

**Comment by monacodelisa** [comment]
I would like to add that @cleancodecraft can be a little bit harsh sometimes, but @jhonacs2 - your efforts are still valued and appreciated

**Comment by monacodelisa** [comment]
> my bad, I will ask more questions in the future! thank you! @cleancodecraft @monacodelisa

no worries

**Comment by monacodelisa** [comment]
@all-contributors please add @jhonacs2  for code, design, security and ideas

**Pull request merged**